### DISCUSSION TITLE
Filing Type

### DISCUSSION THREAD

**Discussion by rezagolestani** [sec-edgar/sec-edgar#179] [General]
Hi, Thanks for the great work, 
I just came across an issue with sec-edgar, when selecting FILING_4 as the form type to be downloaded, it will download any form starting with "4", like form 424B3, 424B5, 424B2. is there any way to filter these out?

below is the code I was using:

`my_filings = Filing(cik_lookup='AMZN', filing_type=FilingType.FILING_4, start_date=datetime(2010, 1, 1))`

**Comment by jackmoody11** [comment]
> Hi, Thanks for the great work, 
> 
> I just came across an issue with sec-edgar, when selecting FILING_4 as the form type to be downloaded, it will download any form starting with "4", like form 424B3, 424B5, 424B2. is there any way to filter these out?
> 
> 
> 
> below is the code I was using:
> 
> 
> 
> `my_filings = Filing(cik_lookup='AMZN', filing_type=FilingType.FILING_4, start_date=datetime(2010, 1, 1))`

Thanks for pointing this out. This is a bug on our side which we should be able to fix. Can you put this in the issue tracker?

**Reply by rezagolestani** [reply]
Thanks for your response, sure, will do,

**Reply by reteps** [reply]
Added to v0.4.0

**Reply by rezagolestani** [reply]
Thanks,
